src/components/screens/consul/Debates/NewDebate.js

```diff
@@ -42,15 +42,15 @@ export const NewDebate = ({ navigation, data, query }) => {
     client: ConsulClient
   });
 
-  const onSubmit = async (val) => {
+  const onSubmit = async (newDebateData) => {
     let variables = {
       id: data?.id,
       attributes: {
         translationsAttributes: {
-          title: val.title,
-          description: val.description
+          title: newDebateData.title,
+          description: newDebateData.description
         },
-        tagList: val.tagList,
+        tagList: newDebateData.tagList,
```

Comment:
these should be secured with conditional chaining i guess to avoid crashes if `newDebateData` is undefined.